keyboards/sck/m0116b/keymaps/m0116/keymap.c

```diff
@@ -0,0 +1,126 @@
+		KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,   KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_ENT,  KC_P7,   KC_P8,   KC_P9,   KC_PPLS,
+		KC_LCTL, KC_A,    KC_S,    KC_D,    KC_F,   KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,                   KC_P4,   KC_P5,   KC_P6,	KC_PMNS,
+		KC_LSFT,          KC_Z,    KC_X,    KC_C,   KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT,          KC_P1,   KC_P2,   KC_P3,   KC_PENT,
+		KC_LCAP, KC_LALT, KC_LGUI, KC_GRV,                   KC_SPC,             LT(1, KC_BSLS), KC_LEFT, KC_UP,   KC_DOWN, KC_RGHT, KC_P0,            KC_PDOT
+),
+/* Keymap Layer 1: (Layer 1) Layer 1
+   * ,---------------------------------------------------------------------------------.
+   * |                     |RESET|                                                     |
+   * `---------------------------------------------------------------------------------'
+   * ,-----------------------------------------------------------. .-------------------.
+   * |   |F1 |F2 |F3 |F4 |F5 |F6 |F7 |F8 |F9 |F10|F11|F12|       | |    |    |    |    |
+   * |-----------------------------------------------------------| |-------------------|
+   * |     |   |   |   |   |   |   |   |   |   |   |   |   |     | |    |    |    |    |
+   * |-----------------------------------------------------|     | |-------------------|
+   * |       |   |   |   |   |   |   |   |   |   |   |   |       | |    |    |    |    |
+   * |-----------------------------------------------------------' |-------------------|
+   * |        |   |   |   |   |   |   |   |   |   |   |          | |    |    |    |    |
+   * |-----------------------------------------------------------' `--------------|    |
+   * |    |    |    |   |    M0116B       |   |   |    |    |    | |         |    |    |
+   * `-----------------------------------------------------------' `-------------------'
+   */
+[1] = LAYOUT_m0116_ansi( /* Layer 1 */
+	                                                          RESET,
+		KC_TRNS, KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_DEL,  KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,                   KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,                   SAFE_RANGE,                   KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS
```

Comment:
```suggestion
		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,                   M0116B,                       KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS
```

Reply:
None of the keymaps  compile when using `M0116B` instead of `SAFE_RANGE`. they all compile perfectly fine, and the sendstring works correctly, when using `SAFE_RANGE`, which is why that is what is in in those spaces

Reply:
corrected